sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/utils/ErrorLogHelper.java

```diff
@@ -215,17 +225,70 @@ public static synchronized File getErrorStorageDirectory() {
         return sErrorLogDirectory;
     }
 
+    /**
+     * A general folder where unprocessed NDK crashes are saved.
+     *
+     * @return a folder name e.g. /lib/files/error/minidump/new
+     */
     @NonNull
-    public static synchronized File getNewMinidumpDirectory() {
+    private static synchronized File getNewMinidumpDirectory() {
+        File errorStorageDirectory = getErrorStorageDirectory();
+        File minidumpDirectory = new File(errorStorageDirectory.getAbsolutePath(), MINIDUMP_DIRECTORY);
+        return new File(minidumpDirectory, NEW_MINIDUMP_DIRECTORY);
+    }
+
+    /**
+     * A session-specific folder where unprocessed NDK crashes are saved.
```

Comment:
session is ambiguous here in the comment, the guid is actually a crash id not a session id. maybe just say one time run specific?